.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
This is going to be a problem because our Android Studios will be replacing it back and forth. @rasprague Could you please remove it from git and add to gitignore? 
Let's resolve this in this PR and then we are ready to merge this since the rest looks good.

Reply:
file reverted, my bad

Reply:
file reverted, my bad